sc/src/main/java/uk/ac/ebi/atlas/home/JsonLatestExperimentController.java

```diff
@@ -0,0 +1,22 @@
+package uk.ac.ebi.atlas.home;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ebi.atlas.controllers.JsonExceptionHandlingController;
+import uk.ac.ebi.atlas.trader.ScxaExperimentTrader;
+import javax.inject.Inject;
+
+@RestController
+public class JsonLatestExperimentController extends JsonExceptionHandlingController {
+    private final LatestExperimentsService latestExperimentsService;
+    @Inject
```

Comment:
In Spring, if a `@Component` class has only one constructor, it applies field injection by default.

[`@RestController` is an annotation that combines `@Controller` and `@ResponseBody`](https://docs.spring.io/spring/docs/4.3.9.RELEASE/javadoc-api/org/springframework/web/bind/annotation/RestController.html). In turn, [`@Controller` is a specialized `@Component`](https://docs.spring.io/spring/docs/4.3.9.RELEASE/javadoc-api/org/springframework/stereotype/Controller.html). So all RestControllers are Spring components.  You don’t need to annotate it with `@Inject`.